src/orderbook.js

```diff
@@ -164,7 +201,15 @@ OrderBook.EVENTS = [
   'offer_changed', 'offer_funds_changed'
 ];
 
-OrderBook.DEFAULT_TRANSFER_RATE = 1000000000;
+OrderBook.DEFAULT_TRANSFER_RATE = Amount.from_json(1000000000);
+
+OrderBook.NOTIFY_THROTTLE_TIME = 4000;
```

Comment:
This only needs to be 250ms or maybe less to catch all the transactions - they all get emitted in a cluster when the ledger closes then nothing happens for several seconds.  My guess is you might get very slightly more calls to compute autobridged offers at 250 vs 4000, but significantly less than not throttling (when the volume is heavy).  As a trader want to be as close to real time as possible on orderbook updates.